projects/ngneat/cashew/src/lib/test/httpCacheInterceptor.spec.ts

```diff
@@ -155,4 +164,37 @@ describe('HttpCacheInterceptor', () => {
     call(request({ cache$: true, bucket$: bucket, key$: 'foo' }), 1);
     expect(bucket.add).toHaveBeenCalledWith('foo');
   }));
+
+  it('should use parameterCodec from request', fakeAsync(() => {
+    const testParam = 'te3/s-+d+_asd:';
+    const expectedParamString = new HttpParams({
+      encoder: new CustomHttpParamsCodec(),
+      fromObject: { testParam }
+    }).toString();
+
+    const handler = {
```

Comment:
you don't need to create your own handler, it is already been created on `beforeEach`, look at the above tests.

Reply:
not sure I understood you correctly, so I removed `const` as it already exists from `beforeEach` test setup

Reply:
What I mean is that you can delete the `handler` entirely since it is already been created on every test inside the `beforeEach`, and then you could get the `call` result from the spy argument, like this: `handler.handle.calls[0][0]`. So you don't have to create your own mock.

Reply:
ah, now I got the memo ^^
changed